pkgs/applications/blockchains/liana/default.nix

```diff
@@ -0,0 +1,80 @@
+{ lib
+, pkgs
+, rustPlatform
+, fetchFromGitHub
+, llvmPackages
+, fetchurl
+, makeDesktopItem
+}:
+let
+  runtimeLibs = with pkgs; [
+    expat
+    fontconfig
+    freetype
+    freetype.dev
+    libGL
+    vulkan-loader
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXi
+    xorg.libXrandr
+  ];
+in
+rustPlatform.buildRustPackage rec {
+  pname = "liana";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "wizardsardine";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-cqvjPa/GXDdFv8XEPJgHfKP/Duzv+jmp1yZ1rlKPfE8=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/gui/Cargo.lock";
```

Comment:
[IFD](https://nixos.wiki/wiki/Import_From_Derivation) is not allowed in nixpkgs. Just put the cargo lock at `pkgs/applications/blockchains/liana/Cargo.lock`.